valid_parentheses.py

```diff
@@ -0,0 +1,15 @@
+class Solution:
+    def isValid(self, s: str) -> bool:

```

Comment:
This is a pretty common problem for interviews… so my recommendation is to explore more solutions for this problem. Maybe some variations of this one